Update Projects.tsx

AkRao2705 opened the pull request:
Added the feature of deleting a specific project by adding functionality to the delete option provided in the dropdown menu. Updated Projects.tsx file:
<img width="527" alt="Screenshot 2025-04-16 at 9 02 50 PM" src="https://github.com/user-attachments/assets/b252c1b8-0ac5-46c3-a07e-6e6b3860bc04" />
<img width="702" alt="Screenshot 2025-04-16 at 9 02 59 PM" src="https://github.com/user-attachments/assets/00d06026-e073-4879-9de6-25b977c14455" />

AkRao2705 committed:
Update Projects.tsx

Added the feature of deleting a specific project by adding functionality to the delete option provided in the dropdown menu. Updated Projects.tsx file:

AkRao2705 committed:
Merge branch 'main' into delete-project-feature

AkRao2705 committed:
Merge branch 'main' into delete-project-feature

Keith3895 commented on src/pages/Projects.tsx:
```diff
@@ -132,12 +132,10 @@ export default function Projects() {
             <ProjectSkeleton key={index} />
           ))
         ) : (
-          // Show actual projects
```
why remove this?

Keith3895 commented on src/pages/Projects.tsx:
```diff
@@ -94,6 +82,18 @@ export default function Projects() {
+
+    try {
+      await axios.delete(`${API_PATHS.PROJECTS}/${projectId}`);
+      setProjects(projects.filter((p) => p.id !== projectId));
```
call the get projects and refresh the list.

Keith3895 reviewed (changes requested):
change the mechanism to show project list post delete

AkRao2705 committed:
Update Projects.tsx

Calls a get function to fetch a list of projects after deletion

KeithAub commented:
re-verify if functionality works.

AkRao2705 committed:
Update Projects.tsx

Updated Projects.tsx to allow file deletion from cloud storage, batch deletion, fetching an updated project list after the deletion of a project.
